config/enabled.yml

```diff
@@ -1002,6 +1002,13 @@ Lint/HandleExceptions:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions'
   Enabled: true
 
+Lint/HashEachMethods:
+  Description: >-
+                 Use Hash.each_key & Hash.each_value instead of
+                 Hash.keys.each & Hash.values.each.
+  StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#hash-each'
```

Comment:
Some more feedback for you here.

- Use backticks around identifiers like class/method names
- It's `Hash#each_key`, not `Hash.each_key` (This notation is commonly used to refer to instance methods. `Hash.each_key` would be a singleton method on `Hash`.)
- Use "and", not "&"